platform/windows/detect.py

```diff
@@ -217,9 +217,21 @@ def configure_msvc(env, manual_msvc_config):
 
     ## Libs
 
-    LIBS = ['winmm', 'opengl32', 'dsound', 'kernel32', 'ole32', 'oleaut32',
+    LIBS = ['winmm','dsound', 'kernel32', 'ole32', 'oleaut32',
             'user32', 'gdi32', 'IPHLPAPI', 'Shlwapi', 'wsock32', 'Ws2_32',
-	    'shell32', 'advapi32', 'dinput8', 'dxguid', 'imm32', 'bcrypt','Avrt']
+        'shell32', 'advapi32', 'dinput8', 'dxguid', 'imm32', 'bcrypt','Avrt']
```

Comment:
Alignment issue (though it preexisted). Might be worth adding the missing space before 'Avrt' and 'dsound' too.